### DISCUSSION TITLE
ID/EAN w zasobie sale/offers

### DISCUSSION THREAD

**Discussion by JurekAndrzejewski** [allegro/allegro-api#5992] [Suggestion - verification]
W związku z nadchodzącą produktyzacją, która do 2024 ma objąć całe allegro może dobrym rozwiązaniem byłoby dodanie id lub EANu produktu do zwrotki w /sale/offers.
![image](https://user-images.githubusercontent.com/88549154/183840117-d2982103-772c-4caa-b3ba-57de228ce211.png)

**Comment by hahyes** [comment]
Generalnie identyfikator produktu jest dostępny odpytując o dane konkretnej aukcji poprzez ten zasób:
![image](https://user-images.githubusercontent.com/37104952/184112767-5b404828-fc54-4ac0-a626-322c75e815a9.png)

**Reply by JurekAndrzejewski** [reply]
Przy kilkuset/kilku tysiącach aukcji jest spoko, mam 100k aukcji do ogarnięcia

**Comment by rtnet-pl** [comment]
+1
opcjonalnie satysfakcjonującym rozwiązaniem byłoby dodanie do GET /sale/offers parametru `gtin` który umożliwiałby wyszukanie ofert pasujących do podanego numeru GTIN.

Cel: aktualizacja ilości i cen ofert z ERPa
